lib/CamDiag.py

```diff
@@ -522,17 +562,8 @@ def perform_analyses(self, baseline=False):
                 msg = "Analysis script file '{}' is missing. Script is ending here.".format(anly_script_path)
                 end_diag_script(msg)
 
-            #Create averaging script import statement:
-            anly_func_import_statement = "from {} import {}".format(anly_func_name, anly_func_name)
-
-            #Run averaging script import statement:
-            exec(anly_func_import_statement)
-
-            #Create actual function call:
-            anly_func = anly_func_name+'(case_name, input_ts_loc, output_loc, var_list, write_html)'
-
-            #Evaluate (run) averaging script function:
-            eval(anly_func)
+            anly_func_args = [case_name, input_ts_loc, output_loc, var_list, write_html]
```

Comment:
It might make sense to add all of the extra `opt` code you have in `create_climo` here as well.  If the code works as expected then it shouldn't modify the current behavior, but will allow one to add arguments to analysis scripts in the future.